modules/islandora_iiif/src/Plugin/views/style/IIIFManifest.php

```diff
@@ -246,7 +264,7 @@ protected function getTileSourceFromRow(ResultRow $row, $iiif_address, $iiif_bas
             ],
           ];
 
-          if ($ocr_url = $this->getOcrUrl($entity, $row, $i)) {
+          if ($ocr_url = $this->getOcrUrl($entity, $this->structuredTextTerm)) {
```

Comment:
`::getOcrUrl()` doesn't actually have a second parameter.

```suggestion
          if ($ocr_url = $this->getOcrUrl($entity)) {
```